src/__tests__/routes/auth.test.js

```diff
@@ -2,31 +2,27 @@ import mongoose from 'mongoose';
 import AuthController from '../../controllers/AuthController';
 import { User, Person } from '../../models';
 import { personData as user } from '../../__mocks__/dummyData';
-import * as statusCodes from '../../constants/statusCodes';
 
 import app from '../../app';
 
 const res = {
-  status() {
-    return this;
-  },
-  json() {},
+  redirect() { }
 };
 
 describe('Social login', () => {
   beforeAll(async () => {
     await User.deleteOne({ email: user.emails[0].value });
-    jest.spyOn(res, 'status');
+    jest.spyOn(res, 'redirect');
   });
 
   it('Should mock the social login controller', async () => {
     await AuthController.socialAuth({ user }, res);
-    expect(res.status).toHaveBeenCalledWith(statusCodes.CREATED);
+    expect(res.redirect).toHaveBeenCalled();
   });
 
   it('Should mock the social login controller for an existing account', async () => {
     await AuthController.socialAuth({ user }, res);
-    expect(res.status).toHaveBeenCalledWith(statusCodes.OK);
+    expect(res.redirect).toHaveBeenCalled();
   });
```

Comment:
@Karlmusingo The two tests above are doing the same thing!

Reply:
done